### ISSUE TITLE
Update TANF-app/doc/Technical-Documentation/

### ISSUE THREAD

**Issue by valcollignon** [raft-tech/TANF-app#1526]
### Context
After #1515 is merged, `TANF-app/doc/Technical-Documentation/` subdirectory markdowns need to be reviewed and updated. 

### ACs:
- [ ] Add README.md w/ TOC
- [ ] Only relevant markdowns are included in this subdir 
- [ ] Update any broken links within markdowns
- [ ] any needed house-keeping to close out #1477

### Tasks:
- [x] Create and update [hackmd](https://hackmd.io/Buvu_yzyQQ-Nne_g8n5KdA) for subdir `README.md` w/ TOC (`adpennington`,`abottoms-coder`)
- [x] audit subdir and list changes needed (i.e. move, add, drop, change) (`abottoms-coder`, `adpennington`)
- [x] Open draft PR with subdirectory updates (`adpennington`)
- [ ] Review and update any broken links (`valcollignon`, `abottoms-coder`)
- [x] drop /docs/ [readme](https://github.com/raft-tech/TANF-app/blob/raft-tdp-main/docs/README.md)
- [x] update main readme ref links

### Notes:
Part of the #1477 documentation update epic

**_Audit results_**:

- Drop
  - [`Prototypes.md`](https://github.com/raft-tech/TANF-app/blob/raft-tdp-main/docs/Technical-Documentation/Prototypes.md) (_most of this is in the archived prototype [repo](https://github.com/HHS/TANF-app-archive) and/or laid out in the [rfq](https://github.com/18F/tdrs-app-rfq/blob/main/Final-RFQ/FINAL-TDRS-software-development-RFQ.md#14-current-prototype)_)
  - [`Acceptance-Criteria-for-Django-admin-console`](https://github.com/raft-tech/TANF-app/blob/raft-tdp-main/docs/Technical-Documentation/Acceptance-Criteria-for-Django-admin-console) (_obsolete but saved these ACs in OneNote for future ref_)
  - [`Technical-considerations.md`](https://github.com/raft-tech/TANF-app/blob/raft-tdp-main/docs/Technical-Documentation/Technical-considerations.md): (_all of this is covered in background, ux, or ADR docs_)
  - [`Configuration-Management-Plan.md`](https://github.com/raft-tech/TANF-app/blob/raft-tdp-main/docs/Technical-Documentation/Configuration-Management-Plan.md): (_we already have one [here](https://hhsgov.sharepoint.com/:w:/r/sites/TANFDataPortalOFA/Shared%20Documents/compliance/System%20Security%20Plan/TDP-Configuration-Management-Plan-ACF-OCIO_20200428%20(cm)_OFA20210513.docx?d=wb2eb9fa3c4fd45e994e6018271c7ba68&csf=1&web=1&e=kM4NyB). what goes into one useful for ref so copied to onenote. the rest is gitflow_)
- Add
  -  `readme.md` that includes TOC
- Change
  - update [`TANF-Git-Workflow.png`](https://github.com/raft-tech/TANF-app/blob/raft-tdp-main/docs/Technical-Documentation/Architecture-Decision-Record/images/TANF-Git-Workflow.png) to `TANF-Git-Workflow.jpg` which is updated version of  git workflow diagram  
  - rename [`jwt-key-rotation.md`](https://github.com/raft-tech/TANF-app/blob/raft-tdp-main/docs/Technical-Documentation/jwt-key-rotation.md) per #1493 
  - [`unit-tests.md`](https://github.com/raft-tech/TANF-app/blob/raft-tdp-main/docs/Technical-Documentation/unit-tests.md) should incorporate relevant commands from #470 
  - link [a11y dos and donts](https://github.com/raft-tech/TANF-app/blob/raft-tdp-main/docs/Technical-Documentation/rafts-accessibility-dos-and-donts.md) in [our-priorities-values-expectations.md](https://github.com/raft-tech/TANF-app/blob/raft-tdp-main/docs/How-We-Work/our-priorities-values-expectations.md#deliverable-4-accessibility)
  
- Move
  - `data-file-downloads.md` into  notes of ADR 013
  - promote [`database documentation`](https://github.com/raft-tech/TANF-app/tree/raft-tdp-main/docs/Technical-Documentation/Database)  into main subdir

**Comment by valcollignon** [comment]
removed from current sprint per @ADPennington. moved into "next sprint backlog/refined and ready to go" to pull into next sprint.

**Comment by abottoms-coder** [comment]
**Drop:**
* `Database/tdp-erd.png`: I've never referenced this.
* `Prototypes.md`: As you said in your audit description.
* `Technical-considerations.md`: The bottom links might be useful somewhere but this document doesn't seem to provide a lot of value.


**Change:**
* `diagrams/tdp-environments.png`: Needs new tdp-project-updates environment
* `Configuration-Management-Plan.md`: I feel this is excessively verbose/long and I'm not sure of the audience. Could be dropped, merged into another doc, rewritten.
* `data-file-downloads.md`: Could this be a subsection of another document?
* `openid-connect.md`: Out of date, could update after NextGen XMS or delete this entirely.
* `unit-tests.md`: I feel like this could be more thorough or we should trash it. We could incorporate the new commands from #470 .

**Other:**
* `TDP-environments-README.md`: I feel like this should be part of the main README and TOC for this folder.

cc: @ADPennington Let me know if you'd like to sync/pair later this week.

**Comment by ADPennington** [comment]
@abottoms-coder for discussion:

- drop list 
- what should go into the TOC in the new [`readme.md`](https://hackmd.io/Buvu_yzyQQ-Nne_g8n5KdA) for this subdir?
- should `data-file-downloads.md` be embedded in notes of ADR 013? 
- should [`a11y dos and donts`](https://github.com/raft-tech/TANF-app/blob/raft-tdp-main/docs/Technical-Documentation/rafts-accessibility-dos-and-donts.md) be moved into [`our-priorities-values-expectations.md`](https://github.com/raft-tech/TANF-app/blob/raft-tdp-main/docs/How-We-Work/our-priorities-values-expectations.md#deliverable-4-accessibility) or referenced therein?

**Comment by ADPennington** [comment]
update `openid-connect.md` as part of #1153 epic (#1561)

**Comment by abottoms-coder** [comment]
Reopened for other changes agreed during tech sync with @ADPennington -- this ticket will need a second or perhaps even third PR.
